tests/tests_suite_test.go

```diff
@@ -48,22 +64,142 @@ var _ = BeforeSuite(func() {
 
 	portForwarder = port_forwarder.New(config, ApiClient.CoreV1().RESTClient())
 
+	transport := http.DefaultTransport.(*http.Transport).Clone()
+	transport.MaxConnsPerHost = 1
+	transport.MaxIdleConnsPerHost = 1
+	transport.DialContext = PortForwardDial
+	transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	TestHttpClient = &http.Client{
+		Transport: transport,
+	}
+
+	originalConfigMap, err = ApiClient.CoreV1().ConfigMaps(DeploymentNamespace).Get(context.TODO(), configMapName, metav1.GetOptions{})
```

Comment:
In the RBAC file, the operator does not have permissions to perform actions on ConfigMaps, how could this work?

Reply:
Because the functional test code is not executed by the operator. The test binary is run by the CI (or local developer) that has full access to the cluster.

Reply:
Cool! Thanks for the explanation!